File: Tarea_7_PY/Usuario.py
class Usuario:
    def __init__(self, nombre):
        self.nombre = nombre
        self.cuentas = []

    def agregarCuenta(self, cuenta):
        self.cuentas.append(cuenta)

    def getNombre(self):
        return self.nombre

    def setNombre(self, nombre):
        self.nombre = nombre

    def getCuentas(self):
        return self.cuentas

    def setCuentas(self, cuentas):
        self.cuentas = cuentas

    def __str__(self):
        return f"Nombre: {self.nombre} - Cuentas: {self.cuentas}"

File: Tarea_7/src/Usuario.java
import java.util.*;
public class Usuario {
    String nombre;
    List<CuentaBancaria> cuentas;

    public Usuario(String nombre) {
        this.nombre = nombre;
        this.cuentas = new ArrayList<>();
    }

    public void agregarCuenta(CuentaBancaria cuenta) {
        cuentas.add(cuenta);
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public List<CuentaBancaria> getCuentas() {
        return cuentas;
    }

    public void setCuentas(List<CuentaBancaria> cuentas) {
        this.cuentas = cuentas;
    }
}
